packages/common/src/store/pages/profile/selectors.ts

```diff
@@ -75,7 +75,7 @@ export const getProfileCollections = createDeepEqualSelector(
       .filter((collection) => {
         if (collection) {
           const { is_delete, _marked_deleted, _moved } = collection
-          return !(is_delete || _marked_deleted) || _moved
+          return !(is_delete || _marked_deleted || _moved)
```

Comment:
@dylanjeffers can you double check this? I'm not sure if it was supposed to be `|| _moved` but that seemed incorrect?